docs/src/main/paradox/google-cloud-pub-sub-grpc.md

```diff
@@ -34,6 +34,15 @@ The table below shows direct dependencies of this module and the second tab show
 
 @@dependencies { projectId="google-cloud-pub-sub-grpc" }
 
+## Binary compatibility
+
+@@@warning
+
+This connector contains code generated from Protobuf files which is bound to @extref:[Akka gRPC $akka-grpc.version$](akka-grpc:). This makes it @extref:[NOT binary-compatible](akka-grpc:/binary-compatibility.html) with later versions of Akka gRPC.
+You can not use a different version of Akka gRPC within the same JVM instance.
```

Comment:
@raboof Would you add anything more to this note?

Reply:
Sounds good to me